### ISSUE TITLE
Add Princeton RAtlas

### ISSUE THREAD

**Issue by adamltyson** [brainglobe/brainglobe-atlasapi#204]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10603261/

**Comment by adamltyson** [comment]
I think this should be a priority, seems like quite a few labs are using it.

**Comment by PolarBean** [comment]
one blocking issue at the moment is the labels appear improperly interpolated. 
![image](https://github.com/user-attachments/assets/d6f2f3ab-31da-4c73-9c05-5eadc1849cf9)
I've raised an issue so hopefully it will be resolved soon.
https://github.com/emilyjanedennis/PRA/issues/10

**Comment by adgibson3** [comment]
Hi all, was able to contact the creator of the above rat atlas. She kindly responded and plans to update with FigShare corrections today.

**Comment by kjungwoo5** [comment]
Hi, could I double check that the annotations that I should use for this is `PRA_WHS_v4_anns` from [their figshare](https://figshare.com/articles/dataset/Princeton_RAtlas_PRA_/24207429?file=51181751)? Would this mean that the structures csv/json file would be identical to the WHS one, and thus I should download the structures hierarchy from the [Waxholme Space Atlas site](https://www.nitrc.org/projects/whs-sd-atlas)?

**Comment by PolarBean** [comment]
yes I believe that's correct. If it helps you can copy much of the structures xml logic from https://github.com/brainglobe/brainglobe-atlasapi/blob/main/atlas_scripts/whs_sd_rat.py

**Comment by kjungwoo5** [comment]
Thank you! I'm already on it with whs_sd_rat, it's been extremely helpful 😉

**Comment by kjungwoo5** [comment]
As discussed with @PolarBean, I've found some issues with the Princeton rat atlas files and have raised them on their Github page https://github.com/emilyjanedennis/PRA/issues/12 

#### Summary: 
Annotations for the main PRA atlas do not match the template:
<img width="640" height="360" alt="Image" src="https://github.com/user-attachments/assets/be874bf6-06ce-4c82-9838-91f10d034621" />
Incorrect interpolation for fPRA and mPRA:
<img width="640" height="360" alt="Image" src="https://github.com/user-attachments/assets/7e04dc32-83b1-40e1-96d6-e77041523f2f" />
Incorrect background correction for fPRA:
<img width="640" height="360" alt="Image" src="https://github.com/user-attachments/assets/a615b9b8-3136-4699-9099-906a48d94325" />

**Comment by PolarBean** [comment]
@kjungwoo5 whats the status of this atlas, are all the problems in the raw data now solved?

**Comment by kjungwoo5** [comment]
> [@kjungwoo5](https://github.com/kjungwoo5) whats the status of this atlas, are all the problems in the raw data now solved?

All the problems in the raw data have been resolved, and the new script should package the atlases correctly. 
The only issue is that we haven't decided on how to deal with atlases which don't have a straight midline and don't provide a hemisphere map. Shall I try emailing Emily and asking if she can provide one?

**Comment by PolarBean** [comment]
no its ok! I think that's on us to resolve in the API by allowing the API to return None or False when no hemisphere map is provided,

**Comment by adamltyson** [comment]
We should make sure that other tools (eg brainreg and brainmapper) can deal with having no hemispheres.
